src/kirby/components/card/card.component.tns.ts

```diff
@@ -0,0 +1,72 @@
+import { Component, OnInit, Input, HostBinding, ViewEncapsulation } from '@angular/core';
+import { Color } from 'tns-core-modules/color';
+import { screen } from 'platform';
+import { EventData } from 'tns-core-modules/data/observable/observable';
+import { FlexboxLayout } from 'tns-core-modules/ui/layouts/flexbox-layout/flexbox-layout';
+
+import { ScssHelper } from '../../scss/scss-helper';
+
+const screenScale = screen.mainScreen.scale;
+declare const CGSizeMake: any;
+declare const android: any;
+
+@Component({
+  selector: 'kirby-card',
+  templateUrl: './card.component.html',
+  styleUrls: ['./card.component.scss']
+})
+export class CardComponent implements OnInit {
+  @Input() title: string;
+  @Input() subtitle: string;
+
+  cardSizeClass = '';
+
+  constructor() { }
+
+  ngOnInit() {
+  }
+
+  onViewLoaded(args: EventData) {
+    const stack = <FlexboxLayout>args.object;
+
+    // A timeout is crap, but try without, fail you will
+    setTimeout(() => {
+      const widthDP = stack.getMeasuredWidth() / screenScale;
+      if (widthDP >= ScssHelper.BREAKPOINT_CARD_L) {
+        this.cardSizeClass = 'card-large';
+      } else if (widthDP >= ScssHelper.BREAKPOINT_CARD_M) {
+        this.cardSizeClass = 'card-medium';
+      } else {
+        this.cardSizeClass = 'card-small';
+      }
+    }, 100);
+    this.addShadow(args);
+  }
+
+  addShadow(args: EventData) {
+    const tnsView = <any>args.object;
```

Comment:
2 ting:

1. Du typecaster til any? Er der ikke en base/interface alle view components har i {N}?
2. Hvorfor ikke sende `const stack`med fra kalderen i stedet for hele `args` objektet?

Reply:
Fixed